File: app/src/main/java/com/listenergao/audioandvideolearning/activity/BaseActivity.java
package com.listenergao.audioandvideolearning.activity;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.listenergao.audioandvideolearning.R;

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_base);
    }
}

File: app/src/main/java/com/listenergao/audioandvideolearning/activity/BaseActivity.kt
package com.listenergao.audioandvideolearning.activity

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.listenergao.audioandvideolearning.utils.ToastUtils

/**
 * @author listenergao
 */
open class BaseActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    fun showToast(msg: String, duration: Int = Toast.LENGTH_SHORT) {
        ToastUtils.toast(msg, duration)
    }
}